content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/saml-sso.md

```diff
@@ -54,4 +183,4 @@ To get help, contact your Customer Success Manager (CSM) or support@cobalt.io. W
 | Ensure that the IdP certificate is accurate. | Copy the IdP certificate once again.<br>• Include `-----BEGIN CERTIFICATE-----` and `-----END CERTIFICATE-----`.<br>• Make sure there are no extra whitespaces. |
 | Check that the organization tokens (**IdP Relay State** in Cobalt) match in the IdP system and Cobalt. Pay attention to quotation marks. | Check that all quotation marks in the organization tokens are straight quotes `" "` and not curly quotes `“ ”`. |
 | Ensure that you added users to the Cobalt platform. | We don’t support user provisioning through an IdP. When leveraging an IdP, make sure that there is an established identity for a user in Cobalt.<br>To establish an identity in Cobalt, a user needs to create a password and sign in to Cobalt. All subsequent sign-ins (after the user identity is established in Cobalt) are initiated through the organization’s IdP. |
-| Ensure that the IdP issuer URL value matches between your identity provider and Cobalt. | The Issuer/Identifier ID in a SAML assertion doesn’t have to be a valid URL path. For [Okta](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Cobalt.html), the IdP issuer URL must start with `http` not `https`.  |
\ No newline at end of file
+| Ensure that the IdP issuer URL value matches between your identity provider and Cobalt. | The Issuer/Identifier ID in a SAML assertion doesn’t have to be a valid URL path. For [Okta](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Cobalt.html), the IdP issuer URL must start with `http` not `https`.  |
```

Comment:
Let's remove the last sentence - For [Okta](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Cobalt.html), the IdP issuer URL must start with `http` not `https`.  | I'm nor sure if that's still accurate. 

Also, all of the values must match, not just the IdP Issuer url. Might be worth updating to be more general, in that all values must match between idp & cobalt

Reply:
This is true - ""The Issuer/Identifier ID in a SAML assertion doesn’t have to be a valid URL path." however, i think it's more important that we ensure all values match.